data/projects.json

```
@@ -68,5 +68,15 @@
         "level": "basic",
         "site": "",
         "tags": "tic-tac-toe, c-projects, game, c++"
+    },
+     {
+        "name": "CodersArena",
+        "display": "Coders Arena",
+        "description": "A (web application) platform to have a coing combat with your friend (30 minutes) with random question from our library and also a platform to create test (MCQ and time based) and also to attend MCQ test.",
+        "author": "fantasy-08",
+        "category": "website",
+        "level": "intermediate",
+        "site": "https://github.com/fantasy-08/CodersArena",
```

Hi @fantasy-08,
This is not a website, left it on an empty string.
```suggestion
        "site": "",
```

---

http://boringcoder.herokuapp.com/
This is the website